internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,70 @@
 package daysteps
 
 import (
+	"fmt"
+	sc "go_sprint_4/internal/spentcalories"
+	"log"
+	"strconv"
+	"strings"
 	"time"
 )
 
 const (
-	// Длина одного шага в метрах
 	stepLength = 0.65
-	// Количество метров в одном километре
-	mInKm = 1000
+	mInKm      = 1000
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("invalid data format")
+	}
+	if strings.Contains(parts[0], " ") {
+		return 0, 0, fmt.Errorf("invalid steps format")
+	}
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("invalid steps format")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Нужно везде поправить

Reply:
Поправил. Но понял только еще про 1 место..(стр. 37)